src/main/java/com/google/codeu/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -1,4 +1,4 @@
-package io.happycoding.servlets;
+package com.google.codeu.servlets;
```

Comment:
This line breaks image uploading for me. When I try to upload an image to the URL, the message only shows a little box with a question mark.

Reply:
I think this is argument is the path of this servlet. Have you tried to run `mvn clean` and then see if that resolves the issue? I feel like that this argument refreshes the path of the servlet and what was stored before disappear.

Reply:
When I change the servlet and run the site I get a (501?) Error saying service unavailable and I can't access the site at all. When I run mvn clean it allows me to actually access the site but that is when I run into the exception when uploading an image. Do images for restaurant reviews load for you?